www/source/index.html.slim

```diff
@@ -27,24 +27,22 @@ header.bg-gradient.margin-top-offset.short-bg.relative
   /! announcement
   .row.columns
     .box-white.shadow.z-20.slide-left
-      h3.large-11.medium-11 Announcing InSpec 3.0
-      p.large-11.medium-11 Plugin system, global attributes, enchanced skip messaging, and more.
+      h3.large-11.medium-11 Announcing InSpec 4
+      p.large-11.medium-11 Numerous bugfixes and new features.
       #expand
         .margin-top-sm.brdr-left.large-11.medium-11
           .margin-left-sm
             h4.t-purple core
-            p New plugin system for InSpec CLI and Train
-            p Enhanced "only_if" skip messaging
-            p Unsupported profiles are now hidden from output
-            p Attributes are now specified in <a href="/docs/reference/profiles/">inspec.yml</a>
-            p Text based severity (e.g. "impact 'low'" )
-            p Tons of security/bug fixes and improvements
+            p Added license acceptance (see below)
```

Comment:
p Inputs system overhaul - more consistent, extensible, and predictable
p Updated AWS-SDK to v3
p Laying the foundation for opt-in telemetry
p Support for the new Chef License
p Add support for `more_permissive_than?` on the File resource
p many other minor bugfixes!